astropy/wcs/wcs.py

```diff
@@ -3586,6 +3588,16 @@ def pixel_scale_matrix(self):
 
         return pccd
 
+    @property
+    def preserve_units(self):
+        """
+        Indicates whether the ``WCS`` class is preserving the original units.
+
+        If `True`, units are always kept as specified, whereas is `False`, some
+        units may in some cases be converted to SI or degrees.
```

Comment:
Don't commit suggested change - it is for illustration only but I would prefer a little bit mode details (for the users) on what this does.
```suggestion
        If `True`, units are always kept as specified in/by header's CUNIT keywords, whereas is `False`, some
        units may in some cases (excluding -TAB axes type) be converted to SI or degrees.
```
"some units in some cases" sounds vague.

Reply:
Done